src/client/components/People/Add.js

```diff
@@ -88,20 +87,22 @@ class AddPeople extends Component {
     validateFieldsAndScroll((err, values) => {
       if (!err && !this.state.emailAlreadyExist) {
         const { prefix, color, preferred, firstName, lastName, email,
-          type, jobType, company, tags, note, phones } = sanitize(values, fields);
+          type, jobType, company, tags, notes, jobDescription, phones } = sanitize(values, fields);
         const newPeople = {
           prefix,
           avatar: { color },
           preferred,
           firstName,
           lastName,
+          name: `${firstName} ${lastName}`,
           type,
           email,
           jobType,
-          companyId: company,
+          companyId: (!company) ? '' : company,
```

Comment:
companyId is not a String neither a company !!!
you shoudl add a propd company not companyId